cmd/agent/dist/conf.d/docker.d/conf.yaml.example

```diff
@@ -1,77 +1,243 @@
+## The agent honors the DOCKER_HOST, DOCKER_CERT_PATH and DOCKER_TLS_VERIFY
+## environment variables to set up the connection to the server.
+## See https://docs.docker.com/engine/reference/commandline/cli/#environment-variables
+
 init_config:
 
+  ## @param docker_root - string - optional - default: /
+  ## Change the root directory for retrieving cgroup statistics. Useful when running inside a
+  ## container with host directories mounted on a different folder.
+  ## Example for the docker-dd-agent container:
+  ## docker_root: /host
+  #
+  # docker_root: /
+
+  ## @param timeout - integer - optional - default: 5
+  ## Timeout in seconds for the connection to the docker daemon
+  #
+  # timeout: 5
```

Comment:
this is legacy, should be removed, it's now configured at the agent level via `docker_query_timeout`